libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/ConfigurationCacheForAndroidIT.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.gradle
+
+import org.jetbrains.kotlin.gradle.util.AGPVersion
+import org.jetbrains.kotlin.test.KotlinTestUtils
+import org.junit.Test
+
+class ConfigurationCacheForAndroidIT : ConfigurationCacheIT() {
+    private val androidGradlePluginVersion: AGPVersion
+        get() = AGPVersion.v4_1_0
```

Comment:
❓  Following Ivan's comment https://github.com/JetBrains/kotlin/pull/3521/files#r450235170 the definition of `AGPVersion.v4_2_0` was added but this still uses `v4.1.0`. Is it intentional? Shouldn't it use `v4.2.0` instead?

Reply:
changed to 4.2.0